management/azure_mgmt_container_service/azure_mgmt_container_service.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
   spec.homepage      = 'https://aka.ms/azure-sdk-for-ruby'
   spec.license       = 'MIT'
 
-  spec.files         = Dir["README.md", "LICENSE.txt", "lib/**/*"]
```

Comment:
Surprised why this was like this originally

Reply:
I think it's because the gem was copied from compute, which had a readme and included this.